packages/contracts-watr/scripts/deployment/deployToken.ts

```diff
@@ -14,7 +15,7 @@ export function deployToken<T extends Token>(tokenArtifact: ArtifactFrom<T>, con
     proxy.transferProxyOwnership(controller)
   })
   const proxy = tokenProxy(implementation, (token) => {
-    token.initialize()
+    token.initialize(generatePrecompileAddress(1983))
```

Comment:
Maybe we could move this `1983` to some `WATR` config file or so? Are we sure it this number will be our `assetId`?